src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -51,9 +51,9 @@ public class EditCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited successfully! Edited participant: %1$s";
+    public static final String MESSAGE_NOT_EDITED = "Provide at least one field to edit!";
```

Comment:
Made use of imperative mood here #CS2101 🔥

Reply:
Could you standardise this language FindCommand (MESSAGE_NO_CRITERIA) as well?

Reply:
Sorry, it seems like the change is made in the other PR. You can ignore this suggestion.